File: Easy/find_the_index_of_the_first_occurrence_in_a_string.php
<?php

class Solution {

    /**
     * @param String $haystack
     * @param String $needle
     * @return Integer
     */
    function strStr($haystack, $needle) {
        $index = strpos($haystack, $needle);
        return intval($index < 0 || $index == "" ? -1 : $index);
    }
}

$solution = new Solution;

print_r($solution->strStr("sadbutsad", "sad"));
print_r($solution->strStr("leetcode", "leeto"));
print_r($solution->strStr("lentest", "test"));

File: Easy/find_the_index_of_the_first_occurrence_in_a_string.py
class Solution(object):
    def strStr(self, haystack, needle):
        """
        :type haystack: str
        :type needle: str
        :rtype: int
        """
        return str(haystack).find(needle)
        
        
print(Solution().strStr("sadbutsad", "sad"))
print(Solution().strStr("leetcode", "leeto"))